<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorSymbolCompletionWidget</class>
 <widget class="QWidget" name="EditorSymbolCompletionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="grpCompleSymbols">
     <property name="title">
      <string>Complete Symbols</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Details</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>11</number>
            </property>
            <property name="topMargin">
             <number>11</number>
            </property>
            <property name="rightMargin">
             <number>11</number>
            </property>
            <property name="bottomMargin">
             <number>11</number>
            </property>
            <item>
             <widget class="QCheckBox" name="chkCompleteBrace">
              <property name="text">
               <string>Complete Braces{}</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkCompleteBracket">
              <property name="text">
               <string>Complete Brackets []</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkCompleteParenthesis">
              <property name="text">
               <string>Complete Parenthesis ()</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkCompleteComments">
              <property name="text">
               <string>Complete Multiline Comments /**/</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkCompleteSingleQuotation">
              <property name="text">
               <string>Complete Single Quotations ''</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkCompleteDoubleQuotation">
              <property name="text">
               <string>Complete Double Quotations &quot;&quot;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkCompleteGlobalInclude">
              <property name="text">
               <string>Complete #include &lt;&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkSkipMathingSymbols">
           <property name="text">
            <string>Skip matching symbols while typing</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="chkRemoveSymbolPairs">
     <property name="text">
      <string>Remove symbol pairs when delete chars</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
